cosmo_tester/test_suites/bootstrap_based_tests/inplace_upgrade_test.py

```diff
@@ -61,7 +61,8 @@ def test_inplace_upgrade(cfy,
 
     cfy.snapshots.download([snapshot_name, '-o', snapshot_path])
     cfy.teardown(['-f', '--ignore-deployments'])
-    hosts._bootstrap_manager(hosts._create_inputs_file(manager))
+    config_file = hosts._create_config_file(manager)
+    hosts._bootstrap_manager(manager, config_file)
```

Comment:
i didn't believe it at first either, but yes, this is the only call site of `._bootstrap_manager`